syncano_sync/__init__.py

```diff
@@ -0,0 +1,15 @@
+import logging
+from contextlib import contextmanager
+
+
+LOG = logging.getLogger('syncano-sync')
+SYNCANO_LOG = logging.getLogger('syncano')
+
+
+@contextmanager
+def mute_log(logger=None):
```

Comment:
I guess syncano library log should probably be a little less verbose by default. It's especially visible in tests of syncano lib which are full of messy logs.